src/tests/directives.md

```diff
@@ -2,7 +2,9 @@
 
 <!-- toc -->
 
-> **FIXME(jieyouxu)** completely revise this chapter.
+<!--
+FIXME(jieyouxu) completely revise this chapter.
```

Comment:
FTR, this was intentional and I think it's more accurate to display this. But maybe elaborate with:

- This chapter is quite outdated at various places.
- The organization is confusing.
- It's potentially better to have per-directive doc comments then backlink to them for details, instead of having them separately listed here.

Reply:
It even includes a specific username to be resolved from. Why should it appear on the UI? Maybe I don't understand, what value does it bring to that page?

Reply:
Makes sense, I opened #2187 to track this instead.